ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -136,6 +136,46 @@ def test_numpy_normal(
     )
 
 
+# poisson
+@handle_frontend_test(
+    fn_tree="numpy.random.poisson",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        shape=st.tuples(
+            st.integers(min_value=1, max_value=2)
+        ),
+        min_value=1,
+        max_value=100
+    ),
+
+    size=st.tuples(
+        st.integers(min_value=1, max_value=10), st.integers(min_value=2, max_value=2)
+    ),
```

Comment:
seems like this is not used so we can remove it :slightly_smiling_face:

Reply:
ahh, I see what happened sorry :sweat_smile: when you merged master `size` was moved to the function below along with the missing parenthesis, you should indeed add this size param, but also pass it to the test, sorry for the confusion!

Reply:
no worries, I was not paying attention when merging 😅. Just added back the `size` argument, thank you again for the review!